pkg/cluster/internal/providers/provider/common/proxy.go

```diff
@@ -60,5 +75,35 @@ func getProxyEnvs(cfg *config.Cluster, getEnv func(string) string) map[string]st
 		envs[NOProxy] = noProxy
 		envs[strings.ToLower(NOProxy)] = noProxy
 	}
-	return envs
+	return envs, nil
+}
+
+// obtains proxy information from docker via docker info against cmder
+func getProxyEnvFromDocker(cmder exec.Cmder) (map[string]string, error) {
+	// get raw fields
+	lines, err := exec.OutputLines(
+		cmder.Command(
+			"docker", "info",
+			"--format",
+			// one per line, upper(key)=value
+			"HTTP_PROXY={{.HTTPSProxy}}\nHTTPS_PROXY={{.HTTPSProxy}}\nNO_PROXY={{.NoProxy}}",
```

Comment:
find typo it should be  `HTTP_PROXY={{.HTTPProxy}}`

Reply:
thanks